content/learn/book/getting-started/setup/_index.md

```diff
@@ -96,42 +97,63 @@ resolver = "2" # Important! wgpu/Bevy needs this!
     ```
-    NOTE: Remember to revert this before releasing your game! Otherwise you will need to include `libbevy_dylib` alongside your game if you want it to run. If you remove the "dynamic" feature, your game executable can run standalone.
-
-* **LLD linker**: The Rust compiler spends a lot of time in the "link" step. LLD is _much faster_ at linking than the default Rust linker. To install LLD, find your OS below and run the given command:
-    * **Ubuntu**: `sudo apt-get install lld`
-    * **Arch**: `sudo pacman -S lld`
-    * **Windows**: Ensure you have the latest [cargo-binutils](https://github.com/rust-embedded/cargo-binutils)
-        ```sh
-        cargo install -f cargo-binutils
-        rustup component add llvm-tools-preview
-        ```
-    * **MacOS**: Modern LLD does not yet support MacOS, but we can use zld instead: `brew install michaeleisel/zld/zld`
-* **Nightly Rust Compiler**: This gives access to the latest performance improvements and "unstable" optimizations
-    
-    Create a ```rust-toolchain.toml``` file in the root of your project, next to ```Cargo.toml```.
-    ```toml
-    [toolchain]
-    channel = "nightly"
-    ```
-    For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
-* **Generic Sharing**: Allows crates to share monomorphized generic code instead of duplicating it. In some cases this allows us to "precompile" generic code so it doesn't affect iterative compiles. This is only available on nightly Rust.
+  - **MacOS**: Modern LLD does not yet support MacOS, but we can use zld instead: `brew install michaeleisel/zld/zld`
+- **Nightly Rust Compiler**: This gives access to the latest performance improvements and "unstable" optimizations
+  Create a `rust-toolchain.toml` file in the root of your project, next to `Cargo.toml`.
+  ```toml
+  [toolchain]
+  channel = "nightly"
+  ```
+  For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
+- **Generic Sharing**: Allows crates to share monomorphized generic code instead of duplicating it. In some cases this allows us to "precompile" generic code so it doesn't affect iterative compiles. This is only available on nightly Rust.
 
 To enable fast compiles, install the nightly rust compiler and LLD. Then copy the contents of [this file](https://github.com/bevyengine/bevy/blob/main/.cargo/config_fast_builds) to `YOUR_WORKSPACE/.cargo/config.toml`. For the project in this guide, that would be `my_bevy_game/.cargo/config.toml`.
 
 If something went wrong, check out our [troubleshooting section](/learn/book/troubleshooting/) or [ask for help on our Discord](https://discord.com/invite/gMUk5Ph).
 
+### Enable Debug Optimizations (Optional)
+
+While it may not be an issue for simple projects, debug builds in Rust can be _very slow_ - especially when you start using Bevy's 2D and 3D features.
```

Comment:
```suggestion
While it may not be an issue for simple projects, debug builds in Rust can be _very slow_ - especially when you start using Bevy to make real games.
```